lib/releases.js

```diff
@@ -0,0 +1,110 @@
+var path = require('path');
+var _ = require('lodash');
+
+var Shipit = module.exports;
+
+/**
+ * Compute the current release dir name.
+ *
+ * @param {object} result
+ * @returns {string}
+ */
+
+var computeReleases = function computeReleases(result) {
+  if (!result.stdout) return null;
+
+  // Trim last breakline.
+  var dirs = result.stdout.replace(/\n$/, '');
+
+  // Convert releases to an array.
+  return dirs.split('\n');
+};
+
+/**
+ * Test if all values are equal.
+ *
+ * @param {*[]} values
+ * @returns {boolean}
+ */
+
+var equalValues = function equalValues(values) {
+  return values.every(function (value) {
+    return _.isEqual(value, values[0]);
+  });
+};
+
+/**
+ * Compute the current release dir name.
+ *
+ * @param {object} result
+ * @returns {string}
+ */
+
+var computeReleaseDirname = function computeReleaseDirname(result) {
+  if (!result.stdout) return null;
+
+  // Trim last breakline.
+  var target = result.stdout.replace(/\n$/, '');
+
+  return target.split(path.sep).pop();
+};
+
+/**
+ * Return the current release dirname.
+ */
+
+Shipit.getCurrentReleaseDirname = function() {
+  var shipit = this;
+
+  return shipit.remote('readlink ' + shipit.currentPath)
+  .then(function (results) {
+    if (!results)
+      return shipit.log('No current release found.');
```

Comment:
I think you should return null here for consistency (null or a string, not undefined or whatever).